src/components/ChartItem/ChartItem.jsx

```diff
@@ -1,15 +1,15 @@
 import RoundImage from "components/Image/RoundImage/RoundImage";
 import styles from "components/ChartItem/ChartItem.module.scss";
 
-export default function ChartItem() {
+export default function ChartItem({ data }) {
   return (
     <div className={styles.chartItemContainer}>
       <div className={styles.content}>
         <RoundImage />
-        <span className={styles.ranking}>1</span>
-        <strong className={styles.name}>르세라핌 채원</strong>
+        <span className={styles.ranking}>{data.rank}</span>
+        <strong className={styles.name}>{data.name}</strong>
       </div>
-      <span className={styles.votes}>204,000표</span>
+      <span className={styles.votes}>{data.totalVotes}</span>
```

Comment:
데이터로 변경 너무 좋습니다!
숫자 부분은 `toLocaleString()`을 활용하여 콤마 표시를 하는 것은 어떨까요?
```suggestion
      <span className={styles.votes}>{data.totalVotes.toLocaleString();</span>
```